e2e/tests/121-Portal/ViewPaProfilePage/OpenEditPaPopUp.spec.ts

```diff
@@ -8,6 +8,7 @@ import { seedPaidRegistrations } from '@121-service/test/helpers/registration.he
 import { resetDB } from '@121-service/test/helpers/utility.helper';
 import { registrationsOCW } from '@121-service/test/registrations/pagination/pagination-data';
 import { test } from '@playwright/test';
+import data from '../../../../interfaces/Portal/src/assets/i18n/en.json';
```

Comment:
Elsewhere you have called this 
```suggestion
import englishTranslations from '../../../../interfaces/Portal/src/assets/i18n/en.json';
```
and I think it would be better to be verbose - data is very vague. This applies to everywhere you are importing `data`.

Reply:
Fixed in all instances